terraform/production/dynamodb.tf

```diff
@@ -0,0 +1,28 @@
+resource "aws_dynamodb_table" "processesapi_dynamodb_table" {
+  name           = "Processes"
+  billing_mode   = "PROVISIONED"
+  read_capacity  = 10
+  write_capacity = 10
+  hash_key       = "id"
+  range_key       = "processName"
```

Comment:
I don't think we are having a range key so please remove (it should only be a hash key with id)